WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -128,7 +128,7 @@ class NotificationsViewController : UITableViewController
 
     override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransitionToSize(size, withTransitionCoordinator: coordinator)
-        tableViewHandler.clearCachedRowHeights()
+        tableViewHandler?.clearCachedRowHeights()
```

Comment:
Let's go ahead and add a comment here as to why we need to check this as an optional.